### DISCUSSION TITLE
Not receiving the GitHub Student Developer Pack benefits

### DISCUSSION THREAD

**Discussion by virtual-designer** [community/community#75961] [GitHub Education]
### Hi everyone,

I have applied for the GitHub Student Developer Pack 3 days ago, but still haven't received any of the benefits. When I check my application status, it shows a green approved tag. But I don't have any benefits at all.
How long will this take?

**Comment by ghost** [comment]
Hey @virtual-designer 

I have also applied for **GitHub Student Pack** 3 days ago & I have also the **Green Approved tag** but it shows to wait because --
_There may be a wait period between verification and access to academic benefits._ 

Wait upto 9 days as per the official message. 

Have Trust on GitHub

**Comment by RPR21** [comment]
@abhinavkumar2369 
I am a verified student since October 1. I am yet to get student benefits. Is there any way to get those benefits? I have raised multiple requests and submitted the information again and again. I wonder if I can ever get those benefits.

**Comment by DoomCoder0** [comment]
I got aproved but won't allowed to open it?

**Comment by minhajhasan-dev** [comment]
same for me! I've applied and 3 days passed! still processing!
